pkgs/applications/emulators/ryujinx/default.nix

```diff
@@ -56,12 +64,27 @@ buildDotnetModule rec {
     pulseaudio
     vulkan-loader
     ffmpeg
+
+    # Avalonia UI
+    libICE
+    libSM
+    libXi
+    libXcursor
+    libXext
+    libXrandr
+    fontconfig
+    glew
   ];
 
   patches = [
     ./appdir.patch # Ryujinx attempts to write to the nix store. This patch redirects it to "~/.config/Ryujinx" on Linux.
   ];
 
+  makeWrapperArgs = [
+    "--set GDK_BACKEND x11"
+    "--set SDL_VIDEODRIVER x11"
```

Comment:
The `SDL_VIDEODRIVER=x11` flag doesn't make much sense considering SDL2 is never used for rendering. We only depend on `SDL2_mixer`, which is used for audio.
```suggestion
```

Reply:
Well, let's tack it back in #187150. Apparently SDL2 is used for input and is thus display protocol specific.

Reply:
Ah, my bad :sweat_smile: